cloudbrain/rt_server/rt_server.py

```diff
@@ -41,17 +43,19 @@ class RtStreamConnection(SockJSConnection):
 
     def __init__(self, session):
         super(self.__class__, self).__init__(session)
-        self.subscribers = {}
+        self.subscribers = recursivedict()
 
 
-    def send_probe_factory(self, metric_name):
+    def send_probe_factory(self, device_id, device_name, metric):
 
         def send_probe(body):
```

Comment:
Consider making some of your methods private with the `_` prefix. This comments is valid for other methods in this file.